sqlmesh/migrations/v0017_fix_windows_seed_path.py

```diff
@@ -0,0 +1,47 @@
+"""Fix seed paths that have a Windows forward slash in them."""
+import json
+
+import pandas as pd
+from sqlglot import exp
+
+
+def migrate(state_sync):  # type: ignore
+    engine_adapter = state_sync.engine_adapter
+    schema = state_sync.schema
+    snapshots_table = f"{schema}._snapshots"
+
+    new_snapshots = []
+
+    for name, identifier, version, snapshot, kind_name in engine_adapter.fetchall(
+        exp.select("name", "identifier", "version", "snapshot", "kind_name").from_(snapshots_table)
+    ):
+        parsed_snapshot = json.loads(snapshot)
+        model_kind = parsed_snapshot["model"]["kind"]
+        if "path" in model_kind:
+            model_kind["path"] = model_kind["path"].replace("\\", "/")
```

Comment:
let's use posix here as it's more robust

Reply:
see above reply